docs/ParameterValidation.md

```diff
@@ -45,13 +45,12 @@ This goals of this document is to define valid/invalid API and Query parameters
   - Valid input range : [1874 , currentYear+5]
     - Status : 200
     - Content-Type: application/json
```

Comment:
Here and throughout, should be consistent on how ":" is used (like after "Name", "Description", "Valid input", etc.

I think a good standard is:
- Name: year
- Description: ...
- Type: ...

So, use ":" instead of "-" (there are a few cases where the dash is used instead), and no space before the colon.